### PR TITLE
UX design clean-up

### PR DESCRIPTION
**Pull request by ngraham76**
# Changes made

- Updated the `Create project` modal to match the Figma design
- Changed font colour of tab labels from light to dark grey, so they no longer appear to be disabled
- Removed extraneous "Shared projects will be displayed on this page" line of text from `Public projects` tab of home page (when no projects are displayed)
- Added missing "Sample projects coming soon" line of text from `Sample Projects` tab of home page
- Added a gap between the search field and filter dropdown

### CODE REVIEW

File: packages/client/hmi-client/src/page/Home.vue [hunk 1]
```diff
@@ -393,7 +396,7 @@ header > section > button {
 }
 
 .filter-and-sort label {
-	padding-right: 0.25rem;
+	padding-right: 0.25]rem;
```
Comment (mwdchang) [inline packages/client/hmi-client/src/page/Home.vue:399]: typo? `0.25]rem`?

Comment (ngraham76) [reply packages/client/hmi-client/src/page/Home.vue:399]: Yup, missed that one @mwdchang  - I'll fix.

File: packages/client/hmi-client/src/page/Home.vue [hunk 2]
```diff
@@ -50,7 +50,7 @@
 				<TabView @tab-change="tabChange">
 					<TabPanel v-for="(tab, i) in projectsTabs" :header="tab.title" :key="i">
 						<section class="filter-and-sort">
-							<div>
+							<div style="padding-right: 16px">
```
Comment (YohannParis) [inline packages/client/hmi-client/src/page/Home.vue:53]:
```suggestion
							<div class="pr-4">
```

**Review by YohannParis — APPROVED** [review]

File: packages/client/hmi-client/src/assets/css/theme/_variables.scss [hunk 3]
```diff
@@ -432,22 +432,22 @@ $tabviewHeaderBorder: solid rgba(0, 0, 0, .12) !default;
 $tabviewHeaderBorderWidth: 0 0 1px 0 !default;
 $tabviewHeaderBorderColor: transparent transparent rgba(0, 0, 0, .12) transparent !default;
 $tabviewHeaderBg: #ffffff !default;
-$tabviewHeaderTextColor: $textSecondaryColor !default;
+$tabviewHeaderTextColor: #1B1D21 !default;
```
[the remark this answers is not in the document]
Comment (ngraham76) [reply packages/client/hmi-client/src/assets/css/theme/_variables.scss:435]: The colour #1B1D21 that I was trying here, I'm going to change that to $textColor - it's 98% similar to this custom dark grey.

File: packages/client/hmi-client/src/components/project/tera-project-configuration-modal.vue [hunk 4]
```diff
@@ -4,73 +4,84 @@
...
+						<div>
+							<label for="new-project-name">Project title* <span>(Required)</span></label>
+							<Textarea
+								id="new-project-name"
+								rows="1"
+								v-model="title"
+								autoResize
+								placeholder="What do you want to call your project?"
+								required
+								:style="{ width: '100%' }"
```
[the remark this answers is not in the document]
Comment (ngraham76) [reply packages/client/hmi-client/src/components/project/tera-project-configuration-modal.vue:19]: I'll remember to use the tailwind CSS conventions next round - thanks.

File: packages/client/hmi-client/src/components/project/tera-project-configuration-modal.vue [hunk 5]
```diff
@@ -4,73 +4,84 @@
...
+							<Textarea
+								id="new-project-description"
+								rows="3"
+								v-model="description"
+								autoResize
+								placeholder="Add a short description"
+								:style="{ width: '100%', resize: 'vertical', maxHeight: '278px', minHeight: '32px' }"
+							/>
+						</div>
+						<!--<div>
```
[the remark this answers is not in the document]
Comment (ngraham76) [reply packages/client/hmi-client/src/components/project/tera-project-configuration-modal.vue:33]: I'll remove it.

File: packages/client/hmi-client/src/components/project/tera-project-configuration-modal.vue [hunk 6]
```diff
@@ -4,73 +4,84 @@
...
+						</div>
+						<div>
+							<label for="new-project-description">Description <span>(Optional)</span></label>
+							<Textarea
+								id="new-project-description"
+								rows="3"
+								v-model="description"
+								autoResize
+								placeholder="Add a short description"
+								:style="{ width: '100%', resize: 'vertical', maxHeight: '278px', minHeight: '32px' }"
```
[the remark this answers is not in the document]
Comment (ngraham76) [reply packages/client/hmi-client/src/components/project/tera-project-configuration-modal.vue:30]: Switching this to tailwinds CSS is giving me a different behaviour, not what was intended. @YohannParis can we chat about this? Maybe we can adjust it to get the intended result.

[the remark this answers is not in the document]
Comment (YohannParis) [reply packages/client/hmi-client/src/components/project/tera-project-configuration-modal.vue]: What is the behavioir issue?

**Review by mwdchang — APPROVED** [review]

**Review by YohannParis — APPROVED** [review]

**Pull request merged**